components/library/icu/patches/source_test_ccapitst.c.patch

```diff
@@ -11,13 +11,11 @@
          { 4, "BOCU-1"},
  
          { 1, "windows-1256"},
-@@ -2830,8 +2830,8 @@
-         { 4, "HZ"},
+@@ -2940,7 +2940,7 @@
  
          { 3, "ISO-2022"},
--        { 3, "ISO-2022-KR"},
+         { 8, "ISO-2022-KR"},
 -        { 6, "ISO-2022-JP"},
-+        { 8, "ISO-2022-KR"},
 +        { 9, "ISO-2022-JP"},
```

Comment:
@Toasterson Actually the culprit for the fail test is here... any idea why?

Reply:
Yes, I probably messed up counting in doing the patch :) We have another patch changing these encodings since some time and I needed to manually update them.

Reply:
The change that needs this change is at https://github.com/OpenIndiana/oi-userland/pull/5629/files#diff-e6c0e6d6a33b92682e8d94ac802ff466L11